app/services/email_service/store/address.rb

```diff
@@ -17,13 +17,28 @@ def get(community_id:)
     from_model(find_model(community_id: community_id))
   end
 
+  def get_all(community_id:)
+    from_models(find_models(community_id: community_id))
+  end
+
   def create(community_id:, opts:)
-    address = Address.call(opts.merge(community_id: community_id))
+    address = Address.call(
+      opts.merge(
+      community_id: community_id,
+      verification_status: :verified # TODO At this point we expect
+                                     # that all saved emails are
+                                     # verified. This will be changed
+                                     # soon.
+    ))
     from_model(MarketplaceSenderEmail.create!(address))
   end
 
   def find_model(community_id:)
-    MarketplaceSenderEmail.where(community_id: community_id).first
+    find_models(community_id: community_id).first
+  end
+
+  def find_models(community_id:)
+    MarketplaceSenderEmail.where(community_id: community_id)
```

Comment:
My feeling about readability is that find_models and find_model add very little in terms of code reuse but by adding one level of indirection make it harder to read this code, i.e. to quickly see what exactly is queried and by which attributes.